charmhelpers/contrib/openstack/context.py

```diff
@@ -2798,6 +2798,18 @@ def __init__(self, bridges_key=None, bonds_key=None, enable_dpdk_key=None,
                 self._mac_pci_address_map[bond_mac.mac] = pci_address
 
         config_bridges = config(bridges_key) or ''
+        # check config_bridges pattern "<str>:<str>","<str>:<str> <str>:<str>" or ""
+        config_bridges_pattern = re.match(
+            r'^((\S+:\S+)( \S+:\S+)*|^$)$',
+            config_bridges
+        )
+        if not config_bridges_pattern:
+            msg = ('Wrong format. '
+                   'Expected format is space-delimited list of key-value pairs.'
+                   'Ex: "br-internet:00:00:5e:00:00:42 br-provider:enp3s0f0"')
+            status_set('blocked', msg)
```

Comment:
The call to `status_set` here would be rather pointless, as you have no idea what the charm implementation might do after the raise. If you want all of the charms to use the same message I guess you could add that through a custom exception or something like that?